website/templates/projectGridTemplates.html

```diff
@@ -49,7 +49,7 @@
                             <fieldset class="project-detail-fields">
                                 <div class="row">
                                     <div class="col-xs-7 col-sm-12 col-md-7 col-lg-8">
-                                        <input class="typeahead" id="input{{theItem.node_id}}" type="text" placeholder="Name of project to find" autofocus>
+                                        <input class="typeahead" id="input{{theItem.node_id}}" style="height: 32px;" type="text" placeholder="Name of project to find" autofocus>
```

Comment:
This seems to be a mistaken commit to here, I was probably working on fixing something else. Current master does not have this change so I would discard this.